### PR TITLE
[14.0] [MIG] partner_time_to_pay

### PR DESCRIPTION
**Pull request by JevinD**
partner_time_to_pay migration from 13.0 to 14.0

### CODE REVIEW

**Comment by bosd** [comment]
Is this still a WIP?

**Comment by JevinD** [comment]
@bosd I am not actively working on this. You can take over if needed.

**Comment by max3903** [comment]
@Urvisha-OSI Can you get this PR fixed and merged?

**Comment by rafaelbn** [comment]
I can review functionally if you ping me @max3903 @Urvisha-OSI

**Review by max3903 — APPROVED** [review]

**Comment by bosd** [comment]
@osi-scampbell Can you fix pre-commit on res_partner.xml ??

File: partner_time_to_pay/views/res_partner.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<odoo>
+    <!-- Copyright 2019 Open Source Integrators
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+    <record id="view_res_partner_form_dtp" model="ir.ui.view">
+        <field name="name">res.partner.form.dtp</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="account.view_partner_property_form" />
+        <field name="arch" type="xml">
+            <field name="property_payment_term_id" position="after">
+                <field name="d2p_ytd" groups="account.group_account_invoice"/>
```
Comment (bosd) [inline partner_time_to_pay/views/res_partner.xml]: plese fix pre-commit by adding a space at closing the tag
`_invoice"/>`
`_invoice" />`
(also on the other lines)

**Review by bosd — CHANGES_REQUESTED** [review]
Please fix pre-commit

**Comment by Freni-OSI** [comment]
@max3903 , It's ready for review. Mr. Travis :green_heart:
cc: @rafaelbn @bosd

**Review by bosd — APPROVED** [review]
LGTM 👍

**Comment by rafaelbn** [comment]
@max3903 what do you think if we add in this module a column (property hide) in invoices view lists with the date of the (last) payment which make full reconcile? Is something do you agree just in case? 😄 

Thanks

**Review by rafaelbn — APPROVED** [review]
Tested in runboat! 👍 

imagen

**Pull request merged**